src/redux/categories/categories.js

```diff
@@ -0,0 +1,20 @@
+import * as actions from '../actionTypes';
+
+// initial state
+const initialState = [];
+
+// reducer
+export default function categoriesReducer(state = initialState, action = {}) {
+  switch (action.type) {
+    case actions.CHECK_STATUS: {
+      return 'Under Construction';
```

Comment:
- Please display the string `Under Construction` as stated in the requirements. Also, this will show that your codes are working
- 
![Capture](https://user-images.githubusercontent.com/63972670/208665221-7240a064-aa2b-4f61-a4d1-67ac168c3361.PNG)